app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -125,22 +124,23 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         RecentTasksStyler.styleRecentTasksEntry(this);
 
-        if (!Utils.isEmulator()) {
-            createGoogleApiClient();
-        }
+        mGoogleApiClient = createGoogleApiClient();
```

Comment:
Getting the app stable without play services is a bigger task. Not for this
PR.
The emulator check is not sufficient to run it on an emulator.
On 26 Jan 2016 11:46, "Said Tahsin Dane" <notifications@github.com> wrote:

> In app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java
> <https://github.com/gdg-x/frisbee/pull/547#discussion_r50819270>:
>
> > @@ -125,22 +124,23 @@ public void onCreate(Bundle savedInstanceState) {
> >          super.onCreate(savedInstanceState);
> >          RecentTasksStyler.styleRecentTasksEntry(this);
> >
> > -        if (!Utils.isEmulator()) {
> > -            createGoogleApiClient();
> > -        }
> > +        mGoogleApiClient = createGoogleApiClient();
>
> With this, did we lose the ability to work with the app without Play
> Services installed? Or doesn't it matter?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/547/files#r50819270>.
>